ceph/rados/core_workflows.py

```diff
@@ -574,6 +575,28 @@ def enable_balancer(self, **kwargs) -> bool:
         log.info(f"the balancer status is \n {out}")
         return True
 
+    def check_file_exists_on_client(self, loc) -> bool:
```

Comment:
This method was already added as part of this PR : https://github.com/red-hat-storage/cephci/pull/2426/files and merged.

Please do a clean rebase of the code and push the changes.

Reply:
Please check, did the rebase and pushed the code.